sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/skills/AddUserSkillTest.java

```diff
@@ -1,67 +0,0 @@
-package org.sunbird.integration.test.user.skills;
```

Comment:
Do not remove function tests for add skill tests as this API is still supported by Sunbird platform though it is deprecated.